docs/commands/rpc-protocol.md

```diff
@@ -829,10 +829,9 @@ Using the optional `json_block` is recommended since v19.0.
   }
 }
 ```
-**Optional "pending", "source", "balance"**
+**Optional "pending", "source"**
 
 _pending, source: version 9.0+_  
-_balance: version 12.0+_  
 Booleans, false by default. Additionally checks if block is pending, returns source account for receive & open blocks (0 for send & change blocks), and returns the balance of the account at the time of the block.
```

Comment:
There is another mention of balance at the end of this sentence.

Reply:
Removed that last mention